app/model/spree/refund_decorator.rb

```diff
@@ -1,6 +1,10 @@
-Spree::Refund.class_eval do
-  def pos_order_id
-    # NOTE(cab): Should we use delegates here?
-    payment.order.pos_order_id
+module Spree
+  module RefundDecorator
+    def pos_order_id
+      # NOTE(cab): Should we use delegates here?
```

Comment:
I am not apposed to comments as long as they bring value to the table. I am not too sure what type of value this provides by leading the next dev down a potential refactor hole. I would also love it if we kept naming (even if it was a nickname) out of the comment. `git blame` or the nicer (aliased) `git source` should be good enough to drudge up who the note was created by.